scripts/veracode/static_analysis.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```

Comment:
I see [this Travis job](https://travis-ci.com/github/Alfresco/acs-packaging/jobs/299049588) is green even though the scan could not be performed. Add `set -e` to exit the script when a command fails (returns a non-zero status).

Reply:
The change has been made.